docs/docs/data-stores/README.md

```diff
@@ -141,3 +141,9 @@ Data stores can hold any JSON-serializable data within the storage limits. This
 * Floats
 
 But you cannot serialize Functions, Classes, or other more complex objects.
+
+### Retrieving a large number of keys
+
+Data stores can retrieve up to {{$site.themeConfig.DATA_STORES_MAX_KEYS}} keys in a single query.
```

Comment:
The user and not the data store is doing the retrieving. Consider changing to:

"You can retrieve up to {{$site.themeConfig.DATA_STORES_MAX_KEYS}} keys from a data store in a single query"